src/select.coffee

```diff
@@ -44,7 +44,12 @@ Ember.Widgets.SelectOptionView = Ember.ListItemView.extend
     # propagate the click event
     # if the dropdown is expanded and we select something, don't propagate
     if @get('controller.showDropdown')
-      @get('controller').send 'hideDropdown'
+      # Keep showing dropdown if it is multi select component
+      unless @get('controller.layoutName') is 'multi-select'
```

Comment:
I refactored it but right now it is not relevant (temporarily) because of the highlighted item bug, I will postpone the showing dropdown in multi-select.